tests/src/test/java/alluxio/shell/command/ChownCommandTest.java

```diff
@@ -39,6 +63,65 @@ public void chown() throws IOException, AlluxioException {
     Assert.assertEquals("user2", owner);
   }
 
+  @Test
+  public void chownValidOwnerValidGroup() throws Exception {
+    setupTestGroupMappingService();
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testFile", WriteType.MUST_CACHE, 10);
+    String newOwner = "alice";
+    List<String> groups = CommonUtils.getGroups(newOwner);
```

Comment:
"alice" will be in a group called "alice" since we are using the IdentityUserGroupsMapping. I guess I can write another fake user-groups mapping service for this purpose.